torch/quantization/fx/quantize.py

```diff
@@ -116,8 +116,9 @@ def insert_observer(
     """
     # In eval mode fixed qparams node are the same as CopyNode and we
     # won't insert observer for them
-    if not model.training and isinstance(observer, torch.quantization.FixedQParamsFakeQuantize):
-        return
+    if not model.training:
```

Comment:
maybe we can remove this then? it's probably added before these checks?

Reply:
this assert was actually pretty useful in the PRs further in the stack, it helped iterate on the code with useful error messages, maybe we can keep this?

Reply:
I see, OK